api/src/resolvers/attendance/utils-attendance.ts

```diff
@@ -20,16 +20,29 @@ export const setBacentaICStatus = async (
   session: any,
   bacentaId: string
 ) => {
-  if (last4Bussing.every((bussing) => bussing < 8)) {
-    await session.run(setBacentaIC, { bacentaId })
+  if (last4Bussing.length === 5) {
+    last4Bussing.shift()
+    if (last4Bussing.every((bussing) => bussing < 8)) {
+      await session.run(setBacentaIC, { bacentaId })
+    }
   }
+  // if (last4Bussing.every((bussing) => bussing < 8)) {
+  //   await session.run(setBacentaIC, { bacentaId })
```

Comment:
Why are you commenting out code. If you don't need it anymore, you should delete it